### ISSUE TITLE
Unable to parse invalid JSON when service unavailable

### ISSUE THREAD

**Issue by andrewhavens** [taxjar/taxjar-ruby#45]
We recently had an error report that occurred when the TaxJar service was unavailable (or the request timed out). Here is the error and stacktrace:

```
JSON::ParserError: 822: unexpected token at '{"error":"Service Unavailable", detail: "Request Timed Out", status: "504"}'
```
<img width="703" alt="Screen Shot 2019-03-22 at 10 51 25 AM" src="https://user-images.githubusercontent.com/446392/54842976-8e6cb000-4c90-11e9-9122-685ab290f329.png">

So it looks like the API (or maybe the gem) is returning invalid JSON (half JSON, half Ruby hash). The JSON should look like this:
```
{"error":"Service Unavailable", "detail": "Request Timed Out", "status": "504"}
```

We're using taxjar-ruby version 2.2.0. I checked the changelog and didn't see anything addressing this, so I assume that it's still an issue.

**Comment by fastdivision** [comment]
Hi @andrewhavens, this looks like a response issue with our API for 504 errors. We've seen this recently from a couple other customers as well. We're investigating the issue and I'll follow up as soon as it's resolved. Thank you!

**Comment by darcy** [comment]
Just got this using version 2.4.1. I'm guessing this is the API.

**Comment by yjukaku** [comment]
Yes, this is still an issue for us as well. @fastdivision , any updates on this error?

**Comment by poops** [comment]
Fixed in #51
